introduction.md

```diff
@@ -0,0 +1 @@
+On my way to becoming a GitHub Master git pushgit pushgit pushgit pushgit pushgit pushgit pushgit pushgit pushgit pushgit pushgit pushgit push!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.